packages/leancode_lint/CHANGELOG.md

```diff
@@ -1,15 +1,22 @@
+# 1.2.0
```

Comment:
I would mention bumping min dart SDK to 2.17

Reply:
Added.

Is setting minSdk to 2.17 a breaking change?

Reply:
I wouldn't say so. It blocks upgrades but it does not break existing code.